coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
```

Comment:
this file should not be here

Reply:
I have added the coverage to the gitignore, which was not done before, I do not know how to remove these files now...